.devcontainer/Dockerfile

```diff
@@ -52,9 +66,13 @@ RUN mkdir -p /usr/local/share/m2 \
     && ln -s /usr/local/share/m2 /home/${USERNAME}/.m2
 
 ARG MAVEN_VERSION=""
-ARG TRINO_VERSION="359"
+ARG TRINO_VERSION="400"
+
 # Install Maven
-RUN su vscode -c "umask 0002 && . /usr/local/sdkman/bin/sdkman-init.sh && sdk install maven \"${MAVEN_VERSION}\"" \
+RUN mkdir -p /tmp \
+    && wget https://dlcdn.apache.org/maven/maven-3/3.8.6/binaries/apache-maven-3.8.6-bin.tar.gz -P /tmp \
```

Comment:
I don't really like that we're latching on to a specific version of Maven here - is there a reason for that? Can we not simply 'apt-get install maven'?

Reply:
done